qctrlopencontrols/driven_controls/predefined.py

```diff
@@ -257,26 +263,50 @@ def _new_bb1_control(
     maximum_rabi_rate: float = 2.0 * np.pi,
     **kwargs
 ) -> DrivenControl:
-    """
-    Wimperis or BB1 control.
+    r"""
+    Creates a BB1 (Wimperis) driven control.
+
+    BB1 driven controls are robust to low-frequency noise sources that perturb the amplitude of
+    the control field.
 
     Parameters
     ----------
-    rabi_rotation : float, optional
-        The total rabi rotation to be performed by the control.
+    rabi_rotation : float
+        The total Rabi rotation :math:`\theta` to be performed by the driven control.
     maximum_rabi_rate : float, optional
-        Defaults to 2.*np.pi
-        The maximum rabi frequency for the control.
+        The maximum Rabi frequency :math:`\Omega_{\rm max}` for the driven control.
+        Defaults to :math:`2\pi`.
     azimuthal_angle : float, optional
-        Defaults to 0.
-        The azimuthal position of the control.
+        The azimuthal angle :math:`\phi` for the rotation. Defaults to 0.
     kwargs : dict
-        Other keywords required to make a qctrlopencontrols.DrivenControls.
+        Other keywords required to make a :py:obj:`DrivenControl`.
 
     Returns
     -------
-    qctrlopencontrols.DrivenControl
-        The driven control.
+    DrivenControl
+        The driven control :math:`\{(\delta t_n, \Omega_n, \phi_n, \Delta_n)\}`.
+
+    Notes
+    -----
+    A BB1 driven control [#]_ consists of four control segments:
```

Comment:
Oh wow, I didn't know `[#]_` worked. Very neat!

Reply:
Yeah, it's pretty cool. Could be an option for the api2 docs too (the converter could just convert all the `[^n]`'s into `[#]_`s).